src/main/scala/ai/privado/tagger/PrivadoSimplePass.scala

```diff
@@ -25,10 +25,10 @@ package ai.privado.tagger
 import ai.privado.metric.MetricHandler
 import ai.privado.model.RuleInfo
 import io.shiftleft.codepropertygraph.generated.Cpg
-import io.shiftleft.passes.SimpleCpgPass
+import io.shiftleft.passes.CpgPass
 import org.slf4j.LoggerFactory
 
-abstract class PrivadoSimplePass(cpg: Cpg) extends SimpleCpgPass(cpg) {
```

Comment:
@pandurangpatil Can you confirm if this is correct? Using CpgPass instead of SimpleCpgPass.

Reply:
@abhstabs is right, keep it as SimpleCpgPass for now. We need to test the changes for this one separately. We will create a separate ticket for this one.